dhctl/pkg/operations/mirror/layouts.go

```diff
@@ -188,7 +193,7 @@ func FillLayoutsImages(
 	layouts.ReleaseChannelImages[mirrorCtx.DeckhouseRegistryRepo+"/release-channel:rock-solid"] = struct{}{}
 }
 
-var digestRegex = regexp.MustCompile(`sha256:([a-f0-9]{64})`)
+var digestRegex = regexp.MustCompile(`@sha256:([a-f0-9]{64})`)
```

Comment:
why do we need `@` here?

Reply:
This regex will match something like `docker.io/something/sha256:6f19499b510098d437de57f962aa43c81ee2e4084c728cac47c37ea4d939fb33` without `@` and it shouldn't